.travis.yml

```diff
@@ -77,3 +94,4 @@ cache:
 before_cache:
   # Delete python bytecode to prevent cache rebuild.
   - find third_party -name "*.pyc" -print -delete
+  
```

Comment:
There are spaces on this last line -- this causes github to show an error. See https://github.com/oppia/oppia/pull/1731/files